src/components/DeltaButton.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <span>
+    <span id="likeButton" v-on:click="Like()"><img src="@/assets/deltaSVG.svg" alt="delta" id="delta" width="20px"/></span>
+    <span id="likes">{{likes}}</span>
+  </span>
+</template>
+<script>
+export default {
+  name: 'DeltaButton',
+  data () {
+    return {
+      likes: 0
+    }
+  },
+  props: ['deltas', 'postId', 'userID'],
+  created: function () {
+    //  this doesn't work??? likes = deltas
```

Comment:
you can do
```
data() {
  return {
    likes: this.deltas
  }
}
```
but this is not recommended. If the prop changes, then this component will be rerendered and the new state will be added. However you can remove the state from here and make the parent change it state, only passing down the state via props.